bot/__main__.py

```diff
@@ -27,7 +28,16 @@
 
 bot.add_check(whitelist_check(channels=WHITELISTED_CHANNELS, roles=STAFF_ROLES))
 
-for ext in walk_extensions():
-    bot.load_extension(ext)
+
+async def load_extensions() -> None:
+    """Async wrapper to load all the extensions."""
+    await bot.unloads_cache.set("bot.exts.evergreen.catify", "https://dummy-msg-link.com")
+    await bot.unloads_cache.set("bot.exts.evergreen.xkd", "https://dummy-msg-link.com")
```

Comment:
```py
    await bot.unloads_cache.set("bot.exts.evergreen.catify", "https://dummy-msg-link.com")
    await bot.unloads_cache.set("bot.exts.evergreen.xkd", "https://dummy-msg-link.com")
```

Are these lines the "testing" code that you mentioned?

Reply:
Yes.